presto-server-main/src/main/java/io/prestosql/server/PrestoServer.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.server;
+
+import com.google.common.base.StandardSystemProperty;
+import com.google.common.primitives.Ints;
+
+import static com.google.common.base.MoreObjects.firstNonNull;
+import static java.lang.String.format;
+
+public final class PrestoServer
+{
+    public static void main(String[] args)
+    {
+        String javaVersion = firstNonNull(StandardSystemProperty.JAVA_VERSION.value(), "null");
+        Integer major = Ints.tryParse(javaVersion);
+        if ((major == null) || (major < 11)) {
```

Comment:
remove redundant braces

Reply:
This is standard formatting. You can find thousands of examples of this in the code base. You can also find plenty without braces. I usually write them for and/or for clarify and visual separation.

Reply:
I checked and we actually have far more without parentheses, so I will change this.